Makefile

```diff
@@ -101,7 +104,12 @@ USAGEFILES = docs/usage.md
 usage: install
 	$(foreach md,$(USAGEFILES),$(USAGE_CMD)$(nl))
 
-.PHONY: check-evy-fmt doc doctest evy-fmt lint toc
+GODOC_CMD = ./scripts/gengodoc.awk -v cmd="evy --help" $(md) > $(O)/out.go && mv $(O)/out.go $(md)
```

Comment:
the `-v cmd=...` is now superfluous. Also the `$(md)` in this originally stood for markdown. Perhaps use `$(filename)` or something? You would also need to update the `md` in the `$(foreach ...)` line below.

Reply:
Fixed, thank you!